mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/61 - Buy UI

Closes #61 

To test:
- Play through introductory scenario
- When told to buy, click Buy toggle on left of screen
- Click workstations tab at top (the help text will keep saying you need to click this tab, ignore this for now and I'll deal with it on another help UI focused branch)
- Click computer
- Click desk in front of Moe/Curly
- Repeat for other person
- Make sure scenario objectives/help text progresses as normal
